Yoast/wordpress-seo
# Deprecate og except image

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* I didn't yet deprecate `article:tag` and `article:section` (waiting to hear whether or not they still need to be ported to the new format)
* I didn't yet deprecate `fb:app_id` because it still needs to be ported to the new format
* `og:image` will also be deprecated in another PR

## Test instructions

This PR can be tested by following these steps:

* In the backend:
    * Fill in `Users` --> `Facebook profile URL` (will be used for `article:author`)
    * Fill in `SEO` --> `Social` --> `Facebook page URL` (will be used for `article:publisher`)
    * Create a post, publish it, and modify it
* In the frontend, the following meta tags should NOT be visible:
    * `article:author`
    * `article:publisher`
    * `article:published_time`
    * `article:modified_time`
    * `og:type`
    * `og:locale`
    * `og:site_name`
* Also, there should be no deprecation notices in the frontend.

## UI changes

## Documentation

## Quality assurance


Fixes #
